WP_Auth0.php

```diff
@@ -240,9 +253,11 @@ public static function get_plugin_dir_url() {
 	}
 
 	public function a0_register_query_vars( $qvars ) {
```

Comment:
what's the purpose of this function?

Reply:
It registers query variables (URL params) with the main `WP_Query` instance that basically runs the whole site.